cql2pgjson/src/main/java/org/folio/cql2pgjson/util/Cql2SqlUtil.java

```diff
@@ -24,16 +24,28 @@
   private Cql2SqlUtil() {
     throw new UnsupportedOperationException("Cannot instantiate utility class.");
   }
-
   /**
    * Convert a CQL string to an SQL LIKE string.
-   * CQL escapes * ? ^ \ and SQL LIKE escapes \ % _.
+   * Signal the underlying function that this is not a quoted String case.
    *
    * @param s  CQL string without leading or trailing double quote
    * @return SQL LIKE string without leading or trailing single quote
    */
   @SuppressWarnings("squid:S3776")  // suppress "Cognitive Complexity of methods should not be too high"
   public static String cql2like(String s) {
+    return cql2String(s,false);
+  
+  }
+  /**
+   * Convert a CQL string to an SQL string.
+   * CQL escapes * ? ^ \ and SQL LIKE escapes \ % _.
+   *
+   * @param s  CQL string without leading or trailing double quote
+   * @param quotedString  Whether this is a quoted String following an = sign
+   * @return SQL string without leading or trailing single quote
+   */
+  @SuppressWarnings("squid:S3776")  // suppress "Cognitive Complexity of methods should not be too high"
+  public static String cql2String(String s, boolean quotedString) {
```

Comment:
The joined method with the quotedString boolean parameter is less readable than two separate methods.

Reply:
I attempted to disable the duplication error in sonar cloud with suppress warning with no success.  I would be open to options.  I do not understand the readability issues as they exist here.  In this case the cql2like is a more restrictive version of the cql2string so it follows logically to call the functions in this way.